### ISSUE TITLE
Changefeed get stuck when network latency always is 100ms

### ISSUE THREAD

**Issue by Tammyxia** [pingcap/ticdc#1866]
## Bug Report

Please answer these questions before submitting your issue. Thanks!

1. What did you do? If possible, provide a recipe for reproducing the error.

- Start changefeed sync task.
- tc add 100ms tidb cluster network latency
- Check changefeed realtime-sync status.

2. What did you expect to see?

3. What did you see instead?

- CF checkpoint don't move on when there's 100ms network latency.
- Is expected behavior that CF cannot work when there's always 100ms network latency?

4. Versions of the cluster

    - Upstream TiDB cluster version (execute `SELECT tidb_version();` in a MySQL client):

        ```
        (paste TiDB cluster version here)
        ```

    - TiCDC version (execute `cdc version`):

        ```
        (paste TiCDC version here)
        ```

**Comment by amyangfei** [comment]
Could you please confirm whether this problem still exists? @Tammyxia

**Comment by Tammyxia** [comment]
Fixed in latest v5.1.0.
